### DISCUSSION TITLE
wagmi/core walletconnect

### DISCUSSION THREAD

**Discussion by Chi111** [wevm/wagmi#2475] [Q&A]
hi
 this is my version @wagmi/core": "0.9.4

```
import { WalletConnectConnector } from '@wagmi/core/connectors/walletConnect';
new WalletConnectConnector({
        chains,
        options: {
          qrcode: true,
        },
});
```

how can i custom qrcode ？

**Comment by Chi111** [comment]
@tmm Excuse me，Can you take a look at this question，thank you

**Reply by tmm** [reply]
Please do not tag me directly in the future and be patient. Check out codebases like [RainbowKit](https://github.com/rainbow-me/rainbowkit) and [ConnectKit](https://github.com/family/connectkit) as they follow this approach, or open a discussion on the WalletConnect repo: https://github.com/orgs/WalletConnect/discussions.

**Reply by Chi111** [reply]
> Please do not tag me directly in the future and be patient. Check out codebases like [RainbowKit](https://github.com/rainbow-me/rainbowkit) and [ConnectKit](https://github.com/family/connectkit) as they follow this approach, or open a discussion on the WalletConnect repo: https://github.com/orgs/WalletConnect/discussions.

thanks
